eclipse-sirius/sirius-components
# [500] Add initial support for conditional styles

### Type of this PR 


### Issue(s)

https://github.com/eclipse-sirius/sirius-components/issues/500

### What does this PR do?

- Adds basic support for conditional styles for NodeDescriptions and
EdgeDescriptions.
- Prevents the creation of a conditional style for the 'style' feature
- Prevents the creation of two EEF pages for conditional styles.

### How to test this PR?

- [x] Manual Test : please specify

1. In a Sirius web project creates a domain model and a view model
2. Defines some domain elements in the domain model
3. Defines how to represents these domain elements in the view model
  - Some NodeDescription or/and EdgeDescription should contain at least a conditional node style
4. In a separated Sirius web project, creates a model "instance" of the domain
5. Creates some elements that can verify the condition of a conditional node style or a condition edge style.
